application/F3DStarter.cxx

```diff
@@ -704,6 +706,7 @@ class F3DStarter::F3DInternals
       interactor.addBinding({ mod_t::NONE, "Drop" }, "add_files_or_set_hdri", "Others", std::bind(docString, "Load dropped files, folder or HDRI"));
       interactor.addBinding({ mod_t::CTRL, "Drop" }, "add_files", "Others", std::bind(docString, "Load dropped files or folder"));
       interactor.addBinding({ mod_t::SHIFT, "Drop" }, "set_hdri", "Others", std::bind(docString, "Set HDRI and use it"));
+      interactor.addBinding( { mod_t::CTRL, "O" }, "open_file", "Others", std::bind(docString, "Open File"));
```

Comment:
I would put this binding before the Drop binding for better ordering

Reply:
Done, I also removed the extra space for better formatting.

Reply:
The doc string should say "Open file dialog" I think.

Reply:
just fixed